### DISCUSSION TITLE
Github pages not updating with error: "Actions jobs for this repository have been disabled by GitHub staff. If you are the repository owner, you can contact support via https://github.com/contact for more information."

### DISCUSSION THREAD

**Discussion by vuoristo** [community/community#23119] [Other Feature Feedback, Questions, & Ideas]
<p>My github pages page <a href="http://vuoristo.github.io" rel="noopener nofollow ugc">vuoristo.github.io</a> has worked well for the past two years. Couple of days ago I pushed an update, which works fine locally, but did not show up on the page. I checked the repo and there is a github actions error with the message “Actions jobs for this repository have been disabled by GitHub staff. If you are the repository owner, you can contact support via <a href="https://github.com/contact" class="inline-onebox" rel="noopener nofollow ugc">GitHub Support</a> for more information.”</p>
<p>Clicking through the link to the support site it’s not at all clear to me how do I open the correct kind of ticket for this. Anyone run into something similar?</p>
<p>Cheers,<br>
Risto</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Yeah, that’s not very obvious. Click on “My Tickets” (top right), and you should see a green “New ticket” button on the page that opens.</p>
